### ISSUE TITLE
the netcoreapp3.0 build is broken

### ISSUE THREAD

**Issue by eerhardt** [dotnet/machinelearning#1495]
Follow the instructions: https://github.com/dotnet/machinelearning/blob/master/docs/building/netcoreapp3.0-instructions.md

You will get a build error:

```
Error	CS0266	Cannot implicitly convert type 'float*' to 'int'. An explicit conversion exists (are you missing a cast?)	Microsoft.ML.CpuMath(netcoreapp3.0)	F:\git\machinelearning2\src\Microsoft.ML.CpuMath\SseIntrinsics.cs	758	Active
```

This is because of #1177 

```C#
                float* pDstEnd = pdst + dst.Length;
                float* pDstCurrent = pdst;
                int destinationEnd = pDstEnd - 4;
```

`pDstEnd` is a `float*`. Subtracting 4 still gives you a `float*`, which is not convertable to `int`.

/cc @jwood803 @tannergooding

**Comment by jwood803** [comment]
@eerhardt Whoops! Sorry about that! I'll get this fixed.

**Comment by eerhardt** [comment]
Resolved by #1497
